bin/index.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+var DraftLog = require('draftlog').into(console).addLineListener(process.stdin)
+var SpeedTestNet = require('../')
+
+const DIM = '\033[2m'
```

Comment:
Chalk detects if color support is available, and perhaps more importantly, has multi-platform support / deals with special cases.